src/commands/init/init.ts

```diff
@@ -26,19 +26,20 @@ const getRepoUrl = (siteInfo) => siteInfo?.build_settings?.repo_url
 
 // @ts-expect-error TS(7031) FIXME: Binding element 'siteInfo' implicitly has an 'any'... Remove this comment to see the full error message
 const logExistingAndExit = ({ siteInfo }) => {
-  log()
-  log(`This site has been initialized`)
-  log()
-  log(`Site Name:  ${chalk.cyan(siteInfo.name)}`)
-  log(`Site Url:   ${chalk.cyan(siteInfo.ssl_url || siteInfo.url)}`)
-  log(`Site Repo:  ${chalk.cyan(getRepoUrl({ siteInfo }))}`)
-  log(`Site Id:    ${chalk.cyan(siteInfo.id)}`)
-  log(`Admin URL:  ${chalk.cyan(siteInfo.admin_url)}`)
-  log()
-  log(`To disconnect this directory and create a new site (or link to another siteId)`)
-  log(`1. Run ${chalk.cyanBright.bold('netlify unlink')}`)
-  log(`2. Then run ${chalk.cyanBright.bold('netlify init')} again`)
-  exit()
+  NetlifyLog.info('')
+  NetlifyLog.message(`This site has been initialized`)
+  NetlifyLog.info('')
+  NetlifyLog.info(`Site Name:  ${chalk.cyan(siteInfo.name)}`)
+  NetlifyLog.info(`Site Url:   ${chalk.cyan(siteInfo.ssl_url || siteInfo.url)}`)
+  NetlifyLog.info(`Site Repo:  ${chalk.cyan(getRepoUrl({ siteInfo }))}`)
+  NetlifyLog.info(`Site Id:    ${chalk.cyan(siteInfo.id)}`)
+  NetlifyLog.info(`Admin URL:  ${chalk.cyan(siteInfo.admin_url)}`)
+  NetlifyLog.info('')
+  NetlifyLog.info(`To disconnect this directory and create a new site (or link to another siteId)`)
```

Comment:
Should this maybe be this instead?

```suggestion
  NetlifyLog.info(`To disconnect this directory and create a new site (or link to another site)`)
```